flink-runtime/src/main/java/org/apache/flink/runtime/jobmaster/slotpool/SlotPool.java

```diff
@@ -323,7 +323,7 @@ public void disconnectResourceManager() {
 			boolean allowQueuedScheduling,
 			Time allocationTimeout) {
 
-		log.debug("Allocating slot with request {} for task execution {}", slotRequestId, task.getTaskToExecute());
+		log.debug("Received slot request [{}] for task: {}", slotRequestId, task.getTaskToExecute());
```

Comment:
I tried to change it for all ids (`SlotRequestID` and `AllocationID`) because they now print `SlotRequestID{1234...}` and `AllocationID{9876...}` in order to make them easier to identify.